.vscode/tasks.json

```diff
@@ -41,7 +41,7 @@
                         "regexp": "Compiling.*?|Compilation .*?starting"
                     },
                     "endsPattern": {
-                        "regexp": "Compiled .*?successfully|Compilation .*?finished"
+                        "regexp": "webpack \\d+\\.\\d+\\.\\d+ compiled|Compilation .*?finished"
```

Comment:
Why was this change necessary? Can you elaborate on this?

Reply:
this seems unrelated to me, @cspotcode can you remove this from the PR?

Reply:
I don't remember entirely, but I think it was in response to some changes to webpack's logging.  Rather than logging "Compiled * successfully" it was logging "webpack <version #> compiled."  The regexp required an update to match.